features/step_definitions/link_pension_webchat_to_pension_and_retirement_pages_steps.rb

```diff
@@ -1,18 +1,39 @@
-Given(/^I visit an article( not| NOT)? within the Pension and Retirement category in my "([^"]*)"$/) do |should_not, language|
-	article =
-    if should_not
-      {
-        'English' => '/en/articles/saving-money-for-christmas',
-        'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
-      }
-    else
-      {
-        'English' => '/en/articles/personal-pensions',
-        'Welsh' => '/cy/articles/pensiynau-personol'
-      }
-    end
+Given(/^I visit "([^"]*)" not within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
+  links = {
```

Comment:
TBH we can make this a private method so that it can get out of the way to make reading the code that little bit easier. Wdyt?

Reply:
Not sure 100% what you are getting at here. Could you show me what you are thinking with some code to explain.